op-supervisor/supervisor/backend/backend.go

```diff
@@ -488,15 +582,28 @@ func (su *SupervisorBackend) CheckAccessList(ctx context.Context, inboxEntries [
 		}
 		entries = remaining
 
-		msgBlock, err := su.checkAccess(acc, executingDescriptor)
+		// Check if message passes time checks
+		if err := executingDescriptor.AccessCheck(su.depSet.MessageExpiryWindow(), acc.Timestamp); err != nil {
+			return err
+		}
+
+		var msgBlockDB eth.BlockID
+		if rpcVerifyAccess {
+			msgBlockDB, err = su.verifyAccessWithRPCInParallel(ctx, executingDescriptor, acc, func(acc types.Access) (eth.BlockID, error) {
+				return su.checkAccessWithDB(acc, executingDescriptor)
+			})
+		} else {
+			msgBlockDB, err = su.checkAccessWithDB(acc, executingDescriptor)
+		}
 		if err != nil {
 			su.logger.Debug("Access-list inclusion check failed", "err", err)
 			return types.ErrConflict
 		}
+
 		// TODO(#14800) add msgBlock to rewind lock
```

Comment:
```suggestion
		// TODO(#14800) add msgBlockDB to rewind lock
```

But I think `msgBlockFromDB` is more descriptive.

Reply:
Fixed in edc6fff1dcb05b4349fc4417d9fa3bb49eb19efa.